<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Guillermo Ortega (ESA)</author>
 <class>stoppingConditionsTrajectoryDialogClass</class>
 <widget class="QDialog" name="stoppingConditionsTrajectoryDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stop conditions (Trajectory)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../iconary/sta-icons.qrc">
    <normaloff>:/icons/TicksButton.png</normaloff>:/icons/TicksButton.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontal2ParametersChoiceTabWidget">
       <item>
        <widget class="QTreeWidget" name="treeWidgetStoppingConditionsTrajectoryToChoose">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerMinimumSectionSize">
          <number>35</number>
         </attribute>
         <column>
          <property name="text">
           <string>Stopping condition choice</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Time</string>
          </property>
          <property name="icon">
           <iconset resource="../../iconary/sta-icons.qrc">
            <normaloff>:/icons/TicksButton.png</normaloff>:/icons/TicksButton.png</iconset>
          </property>
          <item>
           <property name="text">
            <string>Epoch</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1-day</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1-week</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Distance</string>
          </property>
          <property name="icon">
           <iconset resource="../../iconary/sta-icons.qrc">
            <normaloff>:/icons/FlyByTrajectory.png</normaloff>:/icons/FlyByTrajectory.png</iconset>
          </property>
          <item>
           <property name="text">
            <string>to central body</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Periapsis radius</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Angle</string>
          </property>
          <property name="icon">
           <iconset resource="../../iconary/sta-icons.qrc">
            <normaloff>:/icons/maneuvers-rendezvous.png</normaloff>:/icons/maneuvers-rendezvous.png</iconset>
          </property>
          <item>
           <property name="text">
            <string>Ascending node</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1-orbit</string>
           </property>
          </item>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <spacer name="verticalSpacer">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="AddStoppingConditionsTrajectoryPushButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../iconary/sta-icons.qrc">
             <normaloff>:/icons/timeline-double.png</normaloff>:/icons/timeline-double.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="RemoveStoppingConditionsTrajectoryPushButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../iconary/sta-icons.qrc">
             <normaloff>:/icons/timeline-half.png</normaloff>:/icons/timeline-half.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidgetStoppingConditionsTrajectorySelected">
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>0</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>170</number>
         </attribute>
         <attribute name="headerHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Condition</string>
          </property>
          <property name="textAlignment">
           <set>AlignJustify|AlignVCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
          <property name="textAlignment">
           <set>AlignJustify|AlignVCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tolerance</string>
          </property>
          <property name="textAlignment">
           <set>AlignJustify|AlignVCenter</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBoxStoppingConditionsTrajectoryDialog">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBoxStoppingConditionsTrajectoryDialog</tabstop>
 </tabstops>
 <resources>
  <include location="../../iconary/sta-icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBoxStoppingConditionsTrajectoryDialog</sender>
   <signal>accepted()</signal>
   <receiver>stoppingConditionsTrajectoryDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>678</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxStoppingConditionsTrajectoryDialog</sender>
   <signal>rejected()</signal>
   <receiver>stoppingConditionsTrajectoryDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
